src/app/zap-templates/zcl/data-model/chip/webrtc-provider-cluster.xml

```diff
@@ -0,0 +1,144 @@
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+-->
+<!--
+XML generated by Alchemy; DO NOT EDIT.
+Source: src/app_clusters/WebRTC_Provider.adoc
+Parameters: in-progress 
+Git: 0.9-fall2024-234-gf4d359001
+-->
+<configurator xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../../zcl.xsd">
+  <domain name="General"/>
+
+  <enum name="StreamTypeEnum" type="enum8">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item name="Internal" value="0x00"/>
+    <item name="Recording" value="0x01"/>
+    <item name="Analysis" value="0x02"/>
+    <item name="LiveView" value="0x03"/>
+  </enum>
+
+  <bitmap name="WebRTCMetadataOptions" type="bitmap8">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <field name="DataTLV" mask="0x01"/>
+  </bitmap>
+
+  <enum name="WebRTCEndReasonEnum" type="enum8">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item name="IceFailed" value="0x00"/>
+    <item name="IceTimeout" value="0x01"/>
+    <item name="UserHangup" value="0x02"/>
+    <item name="UserBusy" value="0x03"/>
+    <item name="Replaced" value="0x04"/>
+    <item name="NoUserMedia" value="0x05"/>
+    <item name="InviteTimeout" value="0x06"/>
+    <item name="AnsweredElsewhere" value="0x07"/>
+    <item name="OutOfResources" value="0x08"/>
+    <item name="MediaTimeout" value="0x09"/>
+    <item name="LowPower" value="0x0A"/>
+    <item name="UnknownReason" value="0x0B"/>
+  </enum>
+
+  <struct name="ICEServerStruct" apiMaturity="provisional">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item fieldId="1" name="urls" array="true" type="char_string"/>
+    <item fieldId="2" name="username" type="char_string" optional="true"/>
+    <item fieldId="3" name="credential" type="char_string" optional="true"/>
+    <item fieldId="4" name="CAID" type="int16u" optional="true"/>
+  </struct>
+
+  <struct name="WebRTCSessionStruct" apiMaturity="provisional">
+    <cluster code="0x0553"/>
+    <cluster code="0x0554"/>
+    <item fieldId="1" name="ID" type="int16u"/>
+    <item fieldId="2" name="PeerNodeID" type="node_id"/>
+    <item fieldId="3" name="PeerFabricIndex" type="fabric_idx"/>
+    <item fieldId="4" name="StreamType" type="StreamTypeEnum" min="0x00" max="0x03"/>
+    <item fieldId="5" name="VideoStreamID" type="int16u" isNullable="true"/>
+    <item fieldId="6" name="AudioStreamID" type="int16u" isNullable="true"/>
+    <item fieldId="7" name="MetadataOptions" type="WebRTCMetadataOptions" default="0x00" min="0x00" max="0x01"/>
+  </struct>
+
+  <cluster apiMaturity="provisional">
+    <domain name="General"/>
```

Comment:
Think we should be putting these in a Cameras domain

Reply:
I had thought about hand-editing this originally as cameras made sense, but wasn't sure where the value was coming from - seems alchemy gets it from the section header & simply needed Cameras to be added there (done @ https://github.com/project-chip/alchemy/pull/8)